% LaTeX file for resume 
% This file uses the resume document class (res.cls)

\documentclass{res} 

\usepackage[colorlinks=true,linkcolor=blue,urlcolor=blue]{hyperref}

%\usepackage{helvetica} % uses helvetica postscript font (download helvetica.sty)
%\usepackage{newcent}   % uses new century schoolbook postscript font 
\newsectionwidth{0pt}  % So the text is not indented under section headings
\setlength{\textheight}{10.2in} % set text height big enough for box
\topmargin=-.5in       % to start box .5in from top of page
\oddsidemargin=-.5in   % to start box .5in from left of page
\usepackage[margin=0.75in]{geometry}    
\begin{document}
 
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% The following lines define \boxaround, used to draw a box on the page.
% The parameter is the entire text of the resume. Must fit on one page!
%
% \boxaroundhmargin is the left & right margin around the text inside the box.
% \boxaroundvmargin is the top & bottom margin around the text inside the box.
% \boxrulethickness controls thickness of line used to draw the box.
% You can change these 3 things in the lines below:
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


%%%%%%%%%%%%%%%%%%%  End of \boxaround macro %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
%\boxaround{ % put the text on the page inside a box  

\name{Ankit Agrawal\\[12pt]}
\address{\bf Correspondence Address\\S\textbackslash o Rameshwar Prasad Agrawal \\ Beltar Dankeen Gunj, 
Near Punjab National Bank \\ Mirzapur, 231001, Uttar Pradesh, India} 
\address{{\bf Email:} ankitbioinfo@gmail.com\\{\bf Mobile:}  {\bf Mobile:}  +91-89 610 94 901(Brother)\\ {\bf Date:} \today} 





 
\begin{resume}

\section{\sl \underline{Objective}}  % with postscript font \sl produces bold italic
%                           with default font (CM) \sl produces slanted type
%To succeed in an environment of growth and excellence which provides me satisfaction and self development and help me to achieve personal as well as organizational goals
I am seeking a career in teaching as well as scientific research in Life Sciences 

%postdoc position related to subjects in Computational Biology, Bioinformatics, Biophysics, or Systems Biology. I focused on the biophysical aspects of large-scale chromatin architecture and clustering of transcription factor binding sites sequence from ChIP-Seq dataset during my Ph.D. I don't have any constraints for any particular research topic. According to the place and people, I am open to do anything and it should be challenging also.  
      
\section{\sl \underline{Current Job Profile}}
Postdoctoral fellow at Elazar Zelzer lab, Department of Molecular Genetics, Weizmann Institute of Sciences, Israel 
 
\section{\sl  \underline{Education}}
Ph.D. defended in `Computational Biology' on 18th July, 2019 from ``The  Institute of Mathematical Sciences (IMSc), Chennai" affiliated to Homi Bhabha National Institute (HBNI), Mumbai, India \\    
M.Tech. (2012) in Bioinformatics from IIIT Hyderabad, Hyderabad, India \\
M.Sc. (2010) in Bioinformatics from The University of Allahabad, Allahabad, India \\
B.Sc. (2008) in Applied Science from The University of Allahabad, Allahabad India \\
Intermediate (2005) from Board of High School and Intermediate Education, UP India \\
High School (2003) from Board of High School and Intermediate Education, UP India

\section{\sl \underline{Thesis Title}} 
Nuclear Architecture from Chromosomes to Motifs
\href{https://www.researchgate.net/publication/328801683_Nuclear_Architecture_from_Chromosomes_to_Motifs}
{(ResearchGate DOI:10.13140/RG.2.2.22726.32327)}

\section{\sl \underline{Thesis supervisor}}
Gautam I. Menon (IMSc) (Advisor) \\
Rahul Siddharthan (IMSc) (Co-advisor) 

\section{\sl \underline{Google scholar profile}}
Ankit Agrawal \href{https://scholar.google.com/citations?hl=en&user=ey8X7BwAAAAJ}
{(\url{https://scholar.google.com/citations?hl=en&user=ey8X7BwAAAAJ})}

\section{\sl \underline{Github profile}}
@nkit \href{https://github.com/ankitbioinfo}{(\url{https://github.com/ankitbioinfo})}


\section{\sl \underline{Journal Publications}}
\begin{itemize}
\item A. Agrawal, N. Ganai, S. Sengupta and G. I. Menon ``Nonequilibrium Biophysical Processes Influence the Large-Scale Architecture of the Cell Nucleus'' 
\href{https://doi.org/10.1016/j.bpj.2019.11.017}
{Biophysical Journal (2019)}
\item A. Agrawal, S. V. Sambare, L. Narlikar and R. Siddharthan ``THiCweed: fast, sensitive motif finding by clustering big data sets'' 
\href{https://academic.oup.com/nar/article/46/5/e29/4754463}
{Nucleic Acids Research (2018) 46(5), e29} 
\item A. Agrawal, N. Ganai, S. Sengupta and G. I. Menon ``Chromatin as active matter" 
\href{http://iopscience.iop.org/article/10.1088/1742-5468/aa5287}
{Journal of Statistical Mechanics: Theory and Experiment (2017) 014001}
\item A. Agrawal, C. Sarkar, S. K. Dwivedi, N. Dhasmana and S. Jalan ``Quantifying randomness in protein-protein interaction networks of different species: A random matrix approach" 
\href{https://doi.org/10.1016/j.physa.2013.12.005}
{Physica A: Statistical Mechanics and its Applications (2014) 404, 359-367}

\end{itemize}

\section{\sl \underline{Abstract Publications}}
\begin{itemize}
\item A. Agrawal, N. Ganai, S. Sengupta and G. I. Menon ``A First-principles Approach to Large-scale Nuclear Architecture'' 
\href{https://doi.org/10.1016/j.bpj.2017.11.2459}
{Biophysical Journal (2018) 114 (3), 444a-445a}
\end{itemize}


\section{\sl \underline{Dream Challenge Participation and Publications}}
\begin{itemize}
\item We Ankit Agrawal (IMSc), Rahul Siddharthan (IMSc) and Leelavati Narlikar (NCL Pune) as a member of one team participated in ``ENCODE-DREAM in vivo Transcription Factor Binding Site Prediction 2016" challenge, where our task was to predict genome-wide in vivo binding of TFs in a given cell type. We built a classifier that predicts whether a given region is bounded by TF or not. Our team achieved 10th rank out of total 33 team participated. 
\item I participated in ``Dream Idea Challenge Consortium" where the task was to propose an idea related to `data-driven model' rather 
than `model-driven data'. My idea got accepted at first round. Organizers published \emph{``The inconvenience of data of convenience: computational research beyond post-mortem analyses"} paper in \href{https://www.nature.com/articles/nmeth.4457}{Nature Methods Vol. 14, No. 10, 2017} and put all the participants name under ``DREAM Idea Challenge Consortium". 
\end{itemize}




\section{\sl \underline{Plan for next decade}}
\begin{itemize}
\item {\bf Teaching} Plan 
\begin{enumerate}
\item Biostatistics \& Basic Machine Learning: 
    \begin{itemize}
        \item Descriptive statistics
        \item Probability and Probability distributions
        \item Hypothesis testing
        \item Regression and correlation methods 
        \item Supervised and unsupervised learning methods 
        \item Data imputation
    \end{itemize} 
Books: ``Fundamentals of Biostatistics" by Bernard Rosner; ``Probability Theory: The Logic of Science" by E. T. Jaynes; ``Artificial Neural  Networks" by B. Yegnarayana; ``Pattern Classification" by Duda, Hart, and Stork
\item Bioinformatics/Computational Biology/Genomics/Transcriptomics: 
    \begin{itemize}
        \item Comparative genomics: Sequence alignment and dynamic programming, database search, genome annotation, genome assembly, etc 
        \item Coding vs non-coding genes: Hidden Markov Models, gene identification, RNA-seq, RNA folding, etc 
        \item Phylogenomics: molecular evolution and phylogenetics, genetic variation, etc 
        \item Regulation: Gene regulation, regulatory genomics, epigenomics/chromatin states, biological networks, chromatin interactions, etc  
        \item Medical genomics: GWAS, eQTL, cancer genomics, genome editing, personalized medicins, etc.  
        \item Single-cell techniques:  cell state, cell lineage, and cell trajectory 
    \end{itemize}
Books: ``Computational Biology: Genomes, Networks, Evolution" by Manolis Kellis; ``Bioinformatics Algorithms" by Phillip Compeau and Pavel Pevzner; ``Cell and Molecular Biology of Breast Cancer" by  Heide Schatten
\item Biophysics/Molecular Modeling and Simulation: 
    \begin{itemize}
        \item Protein structure and its hierarchy 
        \item Nucleic acids structure: interactions and folding 
        \item Polymers physics (Random walks and diffusion)    
        \item Multiscale simulation methods: Electronic (QM), Atomic/Molecular (Molecular dynamics and Monte Carlo), Mesoscale (Coarse-grain), Macroscale (FEM and FD)  
        \item Thermodynamic ensembles 
        \item Force fields 
        \item Nonbonded computations 
        \item Active matter(out-of-equilibrium systems, \url{https://www.nature.com/collections/hvczfmjfzl})
        \item Introduction of simulation software LAMMPS/NAMD/GROMACS  
    \end{itemize}
Books: ``Molecular Modeling and Simulation" by  Tamar Schlick;  ``Monte Carlo Simulation in Statistical Physics" By Kurt Binder, Dieter W. Heermann; ``Biological Physics" by Philip Nelson,
\item Systems Biology/Mechanobiology/Mathematical Biology/Complex systems:
    \begin{itemize}
        \item Transcription regulation networks (Types of network motifs) 
        \item Principle of robustness (Chemotaxis, Mechanotaxis (Haptotaxis, Durotaxis, Plithotaxis), Kinetic proofreading)
        \item  Stochastic gene expression and stochastic modeling 
        \item Biomolecular self-assembly, self-organization, emergent phenomenon, collective behaviors
%(prediction of microstructure as a function of components, and other conditions)
        \item Pattern formation (autonomous (asymmetric mitosis), inductive (cell-cell signaling), morphogenetic(cell behaviors))
        \item Mechanics of cells and tissues (Key research articles \url{https://www.nature.com/collections/qtrrsfmpdb})
        \item Multiscale agent-based models for multi-cell dynamics (spatial, off-lattice, center-based models, deformable cell models, vertex models)  
        \item Network epidemiology ($R_0$, SIR) 
        \item Synthetic biology (DNA Origami,  Toggle switch, Synthetic ECM) 
        \item Interspecies interactions, the Lotka-Volterra model, and predator-prey oscillations    
        \item Evolutionary dynamics (Genetic algorithm, fitness landscapes, sequence spaces)
    \end{itemize}
Books: ``Nonlinear Dynamics and Chaos" by Steven Strogatz; ``Numerical methods and advanced simulation in biomechanics and biological processes" by Miguel Cerrolaza, Sandra Shefelbine, Diego Garzon-Alvarado; ``Single-Cell-Based Models in Biology and Medicine" by Anderson, Chaplain and Rejniak ; ``Pattern formation in biology, vision and dynamics" by A. Carbone, M. Gromov and P. Prusinkiewicz;  ``Evolutionary Dynamics: Exploring the Equations of Life" by M. A. Nowak;  ``An Introduction to Systems Biology: Design Principles of Biological Circuits" by Uri Alon 


%\item Biomedical Image Processing: Medical data image processing task in ImageJ, MATLAB, Ilastik or scikit-learn
\end{enumerate}

\item {\bf Research} Plan \\ 
\begin{itemize}
\item I will continue my Ph.D. research topic chromatin biophysics where the formation of membrane-less organelle nucleolus by acrocentric chromosome still unclear. Another interesting problem related to lamin proteins that how they change the chromatin organization with binding to some of the chromatin. Recently, it was discovered that chromatin in muscle nuclei exists only on the periphery and it is only possible due to lamin binding proteins. Also, how the dynamics of nuclear membrane affect the dynamics of chromatin, nucleoli other organelles are not much understood.   

\item In the next decade, one of the research problems attract many biologists, physicist and theorists include involvement of mechanical forces in the different biological context, their role in development, physiology, and disease. How the mechanical forces sensed by biological molecules and how they response mechanically in the context of connective/epithelial tissue will be the main mechanism for explaining tissue organization principles. How tissue shape, size, and location are specified during embryogenesis or how the cells migrate collectively and give proper function to tissue if addressed properly then a new understanding of the cause of some birth defects and to the development of strategies to repair them in vivo can be developed in future. There are many ways to modeling collective cell behavior phenomena. The one which I am interested in related to the Voronoi tesselation and vertex model. These models able to preserve the shape of the cell and how cell shape changes responsible for neighboring cell positions are the main ingredients for dynamics. For the experimental data, morphometric cell descriptors for example volume, surface area, curvature, shape index, etc can be learned through any machine learning algorithm to differentiate between normal/disease cell and their local dynamics. Similar principles can apply to cancer metastasis and wound healing also.   



\item We know that the complexity of organism defines by the number of cell types. Prediction of cell type-specific, in vivo transcription factor binding sites, is one of the central challenges in regulatory genomics. Cell type specificity can be regulated at different hierarchical levels of gene expression, pioneer transcription factor, epigenetic marks, and chromatin accessible/inaccessible region. How individual entities get self-assembled and give rise to the functional cell nuclear organization is a complex task which I want to explore in the future. 

\item Cellular heterogeneity present in cell populations is an important factor to distinguish different cell states which is directly related to differentiation, embryogenesis, normal and disease state. One side single-cell transcriptomic technologies are boon to uncover heterogenity but they are too noisy at this stage while on the other side, bulk RNA-seq or GRO-seq methods only measure the average behavior. It is often that the changes in gene expression only reflect the changes in cell-type composition rather than changes in cell states. Estimating the cell-type composition from bulk RNA-seq/GRO-seq data is a computationally challenging task. A widely popular deconvolution method used for such a task but highly cell type and method dependent and has many limitations. I want to explore this problem with the help of my computer science and mathematics colleagues. Also, there is no method exist up to my knowledge to link the connection between gene expression and spatial positioning of genes. If possible then I will also look at these problems in the future too.   


\item  It seems to me, the background of all phenomenon appears to lies in the invisible organization. My future goal is to find such an invisible organization in any biological system.  Overall, my wish to be an expert on topics like chromatin organization (in normal and cancerous cell types, as well as different cell cycle conditions), collective cell migration in a tissue (epithelial vs connective), and data analysis of high-throughput data (from ENCODE, and 3d tif images of cell/nuclei, etc.) through machine learning and molecular dynamics methods. Also if time permits then I want to participate in different Dream-challenges to solve computational challenges.     
  
%\url{http://www.socr.umich.edu/projects/3d-cell-morphometry/}
%\url{https://www.encodeproject.org/}

\end{itemize}




%\section{\sl  Research Experience}
%I had my research experience on networks and graph spectra at IIT Indore during Jan 2013 to Jun 2013\\
%Working experience on nonlinear dynamics, machine learning and neurodynamics at IIIT Hyderabad during Aug 2011 to Apr 2012\\
%Programming experience in Python

\section{\sl \underline{Research Skills}}
%\begin{ncolumn}{2}
%{\it Technical} 
%\end{ncolumn}\\ 
Programming Language: C, C++, Python (NetworkX, Numpy, MatplotLib), LaTeX, and MATLAB\\
Simulation Software: LAMMPS (Large-scale Atomic/Molecular Massively Parallel Simulator) \\
Sequencing Software: NGS/Bioinformatics related tools (bedtools, deeptools, meme etc.) \\
Visualization Software: VMD, Povray, MATLAB, Basic Blender \\
Image Processing: ImageJ, Ilastik, scikit-learn, MATLAB 

%\begin{ncolumn}{2}
%{\it Professional} 
%\end{ncolumn}\\ 

%Alignment Tools: BLAST, FASTA, ClustalW package, VEGA, VAST\\
%Software: MEGA 4, SYSTAT, COPASI Biochemical network analysis, Cytoscape \\
%Package: Xmgrace, Latex, and TCL  

\section{\sl  \underline{Invited Speaker}} 
\begin{itemize}
\item Invited talk on ``A Large Scale Model of Nuclear Architecture" at discussion meeting of ``Aspects of gene and cellular regulation"  held on August 2016, at {\it IMSc Chennai}, India  
\item Invited talk on ``A Large Scale Model of Nuclear Architecture" at 3rd BSSE Annual Research Symposium" on Computational Bioengineering held on January 2017, at {\it IISc Bangalore}, India
\item I gave a talk on `` Nuclear Architecture from Chromosomes to Motifs'' at Bar-Ilan University, Israel on 27 June, 2018
\item I gave a talk on `` Nuclear Architecture from Chromosomes to Motifs'' at Weizmann institute of science, Israel on 28 June, 2018
\item I gave a talk on `` Nuclear Architecture from Chromosomes to Motifs'' at Tel Aviv University, Israel on 3rd July, 2018
\item I gave a talk on `` Nuclear Architecture from Chromosomes to Motifs'' at Technion - Israel Institute of Technology, Israel on 4th July, 2018
\end{itemize}

\section{\sl \underline{Conferences and Workshops}} 
\begin{itemize}
\item Poster presentation in ``TAU-ESPCI international summer school on self-organization and self-assembly: from physics and chemistry to biology" held in {\it Tel Aviv University}, Israel from September 8-12, 2019
\item Attended "Methods and Problems in BioImaging Workshop" in {\it Weizmann Institute of Science}, Israel on June 24th, 2019
\item Gave a talk on `Higher-order Chromatin Architecture' in ``Computaional Biology Annual Meeting" on March 22, 2018 at {\it IMSc}, Chennai
\item Posters presentation in EMBO meeting ``The nucleosome: From atoms to genomes" from August 30 to September 1, 2017, in {\it EMBL Heidelberg}, Germany
\item Poster presentation in discussion meeting on ``Emergence and Evolution of Biological Complexity" was organized at {\it NCBS Bangalore}, India from February 4-6,  2017 
\item Attended ``The Interface of Biology and Theoretical Computer Science" meeting which held on December 19-21, 2016, at {\it NCBS Bangalore}, India
\item Poster presentation in discussion meeting on ``Conflict \& Cooperation in Cellular Populations (CCCP)" was organized at {\it NCBS Bangalore}, India from October 16-19,  2016 
\item Poster presentation in discussion meeting on ``Mechanical Forces in Cell Biology Information at the Cell \& Tissue Scale" was organized at {\it NCBS Bangalore}, India from October 4-6, 2016  
\item Poster presentation in ICTS-ICTP program ``Winter School on Quantitative Systems Biology" which held on December 2015, at {\it ICTS Bangalore}, India
\item Attended ``Advanced Workshop on Interdisciplinary View on Chromosome Structure and Function" which held on September 2014, at {\it ICTP, Trieste} Italy 
\item Attended ``NCNSD (National Conference on Nonlinear Systems and Dynamics)" which held on July 2012, at {\it IISER Pune}, India 
\item Participated in the interaction session of ``Science Conclave: A Congregation of Nobel Prize Winners" organized by {\it IIIT, Allahabad}, India in December 2009 
\end{itemize}




\section{\sl  \underline{Achievement}}
\begin{itemize}
\item Ph.D. fellowship provided by Government of India ``Department of Atomic Energy" 
\item Achieved 10th all India rank in ``Bioinformatics National Certificate Exam" (BINC-2013) conducted by JNU, DBT India 
\item Got EMBO Travel grant for attending the conference ``The nucleosome: From atoms to genomes" from August 30 to September 1, 2017, in {\it EMBL Heidelberg}, Germany
\item Got half funding for attending the workshop ``Advanced Workshop on Interdisciplinary View on Chromosome Structure and Function" which held on September 2014, at {\it ICTP, Trieste} Italy   
\end{itemize}
 
\section{\sl  \underline{Coursera, edX certificates}}
\begin{itemize}
\item Accomplishment certificate of ``Introduction to Artificial Intelligence" offered by Sebastian Thrun and Peter Norvig, passed with a score of 67.7\% 
\item Accomplishment certificate of ``Synapses, Neurons and Brains" offered by The Hebrew University of Jerusalem through coursera, with a score of 87.0\% 
\item Accomplishment certificate of ``Programmed cell death" offered by LMU through coursera, with a score of 52.8\% 
\item Accomplishment certificate of ``Case study: ChIP-seq data analysis" offered by HarvardX through edX, with a score of 71\%  
\end{itemize}




\end{itemize}


%\section{\sl \underline{Personal}}
%Date of Birth : 14th February, 1989\\
%Sex : Male\\
%Personal Email: ankitbioinfo@gmail.com \\
%Date: \today   \\
%Marital Status : Single

\section{\sl  \underline{Details of Projects}} Projects are ordered according to my knowledge and interest. \\

\begin{enumerate}
\item Project: A first-principles approach to large-scale nuclear architecture \\
 Status: Completed  \\ 
 Members: \emph{ Ankit Agrawal}, Nirmalendu Ganai, Surajit Sengupta (TIFR Hyderabad) and Gautam I Menon (IMSc Chennai) \\
 Summary: Model approaches to nuclear architecture have traditionally ignored the biophysical
consequences of ATP-fueled active processes acting on chromatin. However, transcription-coupled
activity is a source of stochastic forces that are substantially larger than the Brownian
forces present at physiological temperatures. Here, we describe large-scale nuclear architecture in interphase human cell nuclei that incorporates cell-type-specific active processes. We reproduce, through computer simulations, differential positioning of euchromatin and heterochromatin, the territorial organisation of
chromosomes, the non-random locations of chromosome territories and observations of both
gene-density-based and size-based radial positioning schemes for chromosomes. The model
predicts positional distributions, shapes and overlaps for individual chromosomes. \\




\item Project: Clustering of motifs in ChIP-Seq data \\
 Status: Completed  \\
 Members: \emph{Ankit Agrawal}, Snehal V. Sambare, Leelavati Narlikar (NCL Pune) and Rahul Siddharthan (IMSc Chennai) \\
 Summary: Most Ab initio motif finding existing algorithms do not scale with large-datasets or fail to report many motifs to those which are associated with cofactors or does not bound to DNA directly or present only in a small fraction of sequences.
So here we develop a program THiCweed which uses divisive hierarchical clustering approach based on sequence similarity to
find the motifs in each cluster. THiCweed thus goes beyond traditional motif finding to give new insights into genomic transcription factor-binding complexity. \\


\item Project: Chromatin as active matter   \\
 Status: Completed  \\
 Members: \emph{Ankit Agrawal}, Nirmalendu Ganai, Surajit Sengupta (TIFR Hyderabad) and Gautam I Menon (IMSc Chennai) \\
 Summary: Active matter models describe a number of biophysical phenomena at the cell and tissue scale. Such models explore the macroscopic consequences
of driving specific soft condensed matter systems of biological relevance out of equilibrium through ‘active’ processes. Here, we describe how active matter
models can be used to study the large-scale properties of chromosomes contained within the nuclei of human cells in interphase. \\


\item Project: Network studies of protein-protein interaction netwrok inframwork of random matrix theory  \\ 
 Status: Completed  \\
 Members: \emph{Ankit Agrawal}, Camelia Sarkar, Sanjiv Kumar Dwivedi, Nitesh Dhasmana and Sarika Jalan (IIT Indore) \\ 
 Summary: Here, we analyze protein-protein interaction networks for six different species under the
framework of random matrix theory. Nearest neighbor spacing distribution of the
eigenvalues of adjacency matrices of the largest connected part of these networks emulate
universal Gaussian orthogonal statistics of random matrix theory. We demonstrate
that spectral rigidity, which quantifies long range correlations in eigenvalues, for all
protein-protein interaction networks follow random matrix prediction up to certain ranges
indicating randomness in interactions. After this range, deviation from the universality
evinces underlying structural features in network. \\


\end{enumerate}







\end{resume}

%\vfill} %    end the material being boxed.
\end{document}


